dist/munin/obs_srcserver_packages

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh
+
+case $1 in
+  config)
+       cat <<'EOM'
+graph_title OBS Source Server: The number of packages
+graph_vlabel Source packages
+graph_category obs
+graph_order numpackages
+numpackages.label obs_src_server_number_of_packages
+numpackages.info The number of source packages in obs source server
+numpackages.type GAUGE
+EOM
+       exit 0;;
+esac
+
+num_numpackages=`ls /srv/obs/sources/ | wc -l`
```

Comment:
This will potentially take a _very_  long time and will produce _very_ much IO. I'm pretty sure that you can count this in a more efficient way, for instance from the frontend. 

`cd /srv/www/obs/api/ && rails runner -e production "puts Package.count"`